financial-templates-lib/ExpiringMultiPartyClient.js

```diff
@@ -93,27 +115,37 @@ class ExpiringMultiPartyClient {
       this.sponsorAddresses.map(address => this.emp.methods.getCollateral(address).call())
     );
 
-    const nextUndisputedLiquidations = [];
-    const predisputeState = "1";
-    const currentTime = Date.now() / 1000;
```

Comment:
I think the EMP Client should be using the same concept of time as the contracts do (i.e. block time), otherwise we can get into some tricky to debug situations where real time differs by block time